app/data/census_household.json

```diff
@@ -889,15 +906,16 @@
                 {
                     "type": "interstitial",
                     "id": "household-member-begin",
-                    "title": "{{ answers.person_full_name[group_instance] }}",
+                    "title": "household member begin",
                     "sections": [
                         {
-                            "title": "{{ answers.person_full_name[group_instance] }}",
+                            "title": "{{ [answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name }}",
                             "id": "household-member-begin-section",
                             "questions": [
                                 {
                                     "id": "household-member-begin-question",
                                     "title": "1. The following questions are for:",
+                                    "description": "<div style='background-color: #efefef; padding: .7em; display: inline-block; border: 2px solid #ccc; font-weight: bold;'>{{ [answers.first_name[group_instance], answers.middle_names[group_instance], answers.last_name[group_instance]] | format_household_name }}</div>",
```

Comment:
Actually I'm mixjng this up with word emphasis for UKIS and the tech debt card we added about removing HTML from the schemas. In other words not sure where this fits in; can we review in the morning?